src/components/PhoneButton.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { ReactChild } from 'react';
+import {
+  Image,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  Linking,
+  Platform,
+} from 'react-native';
+import { theme } from '../styles/colors';
+
+function dialCall() {
+  let phoneNumber = '';
+
+  if (Platform.OS === 'android') {
+    phoneNumber = 'tel:${7344873387}';
+  } else {
+    phoneNumber = 'telprompt:${7344873387}';
```

Comment:
This is actually trying to dial "${7344873387}"

Reply:
Isn't that their phone number? 734.487.3387?

Reply:
Yeah, the problem is the `${` and `}`

`${}` is used for variable interpolation in [\`template literals\`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
Took them out!